frontend/src/app/features/bim/ifc_models/bcf/list-container/bcf-list-container.component.html

```diff
@@ -1,37 +1,39 @@
-<div class="result-overlay"
-     *ngIf="(showResultOverlay$ | async) && showTableView"></div>
+<div class="op-bcf-list-container">
```

Comment:
The addition element causes the layout to break: The card view on the right is not scrollable any more and the toolbar is not visible. I also don't think we really need the container. Instead, I would suggest the following structure:

```
 <div class="*op-bcf-list--result-overlay"
          *ngIf="..."></div>

     <!-- TABLE + TIMELINE horizontal split -->
     <div class="op-bcf-list---work-packages-split-view--tabletimeline-container"
          [ngClass]="..."
          *ngIf="...">
      ....
     </div>

     <!-- GRID representation of the WP -->
     <div *ngIf="..."
          class="op-bcf-list--card-view-container"
          [ngClass]="..." >
       ...
     </div>

     <!-- Footer -->
     <div class="..."
          *ngIf="...">
         ...
     </div>
```